apps/web/src/components/digest-demo-flow/DigestDemoFlowProvider.tsx

```diff
@@ -28,12 +49,13 @@ export const DigestDemoFlowProvider = ({
   isReadOnly: boolean;
   templateId?: string;
 }) => {
-  const [{ isRunningDigest, triggerCount, digestInterval, emailsSentCount }, setState] = useState({
-    isRunningDigest: false,
-    triggerCount: 0,
-    digestInterval: 10,
-    emailsSentCount: 0,
-  });
+  const [{ isRunningDigest, triggerCount, digestInterval, emailsSentCount, hoveredHintId }, setState] =
```

Comment:
Added `hoveredHintId` to the digest demo flow context which says which step popover has hovered. It is used to calculate the hint opacity later on.